src/test/java/Homework19.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest{
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        //Login
+        provideEmail("salma.ayad.87@gmail.com");
+        providePassword("te$t$tudent");
+        clickSubmitBtn();
+        clickOnPlaylist();
+        clickOnDeletePlaylist();
+        Thread.sleep(2000);
```

Comment:
you can use instead sleep wait example 
wait.until(ExpectedConditions.invisibilityOfElementLocated(YouElement);